scripts/automation/trex_control_plane/server/outer_packages.py

```diff
@@ -13,7 +13,15 @@
 
 def _import_server_modules():
 
-    cpu_vendor = 'arm' if os.uname()[4] == 'aarch64' else 'intel'
+    march = os.uname()[4]
+
+    if march == 'aarch64':
+        cpu_vendor = 'arm'
+    elif march == 'x86_64' or march == 'i386' or march == 'i686':
+        cpu_vendor = 'intel'
+    else:
+        assert(cpu_vendor, 'Unknown CPU architecture')
```

Comment:
This assert will never raise Exception

Reply:
Changed the assert to "raise Exception()".  Unsure how to test the code.  If I remove the ppc64le support neither the unit test nor the functional tests documented in [how-to-build-trex](https://github.com/cisco-system-traffic-generator/trex-core/wiki#how-to-build-trex) fail.